### PR TITLE
Open proposal: Miner analyzer and stats

### PR DESCRIPTION
**Pull request by morrigan**
Web solution that gives complete insight to miners about their node status and performance.

### CODE REVIEW

**Comment by morrigan** [comment]
This PR should be labeled as: `category:app-dev` and `proposal-type:open`

File: open-grant-proposals/open-proposal-miner-analyzer.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+- [Belma Gutlić](https://www.linkedin.com/in/belmagutlic/)
+- [Marin Petrunić](https://www.linkedin.com/in/mpetrunic/)
+
+## Team Website
+
+[https://nodefactory.io](https://nodefactory.io)
+
+## Relevant Experience
+
+You can find some of our previous work in the [portfolio](https://www.nodefactory.io/Portfolio-NodeFactory-v3.pdf). Currently, we are also working on a similar application for Eth2 validators called ChainGuardian that has been funded by MolochDAO.
```
Comment (pooja) [inline open-grant-proposals/open-proposal-miner-analyzer.md]: I get a 404 when trying to access the portfolio. Could you provide a new link?

File: open-grant-proposals/open-proposal-miner-analyzer.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    * inspection of deal's state
+    * payment redeemal status
+* utilization and disk space occupancy,
+* status of pledged collateral (past slashings).
+
+Note that this functionalities may slightly change as we are open to adapting to user's needs or Filecoin network changes discovered during the development process. 
+
+## Development Roadmap
+
+### Milestone 1
```
Comment (pooja) [inline open-grant-proposals/open-proposal-miner-analyzer.md]: Would this project make sense to start after testnet? Our APIs, protocol, and implementations will change significantly between now and then.

File: open-grant-proposals/open-proposal-miner-analyzer.md (hunk 1 above)
Comment (morrigan) [reply open-grant-proposals/open-proposal-miner-analyzer.md]: Sorry about that, we are working on our website for the last few days. You can access it now using the same link.

File: open-grant-proposals/open-proposal-miner-analyzer.md (hunk 2 above)
Comment (morrigan) [reply open-grant-proposals/open-proposal-miner-analyzer.md]: It is certainly something that we wanted to go through together with you to know what changes we can expect and plan accordingly. However, we think that milestone 1 (and even part of milestone 2) is not necessarily affected by those changes as it is more about all the use cases and UI/UX design.

**Comment by morrigan** [comment]
> Thanks for the proposal!

Thank you for the comments! Let me know if you have any additional questions. We would be happy to discuss everything on a call too.

File: open-grant-proposals/open-proposal-miner-analyzer.md (hunk 2 above)
Comment (pooja) [reply open-grant-proposals/open-proposal-miner-analyzer.md]: Understood, thanks for clarifying.

**Comment by pooja** [comment]
We've finished the wave 1 grant selection process. Can you please email devgrants@filecoin.org so we can share our decision and next steps, if any? In your email, please include a link to your proposal PR so we know who we're talking to!

Thanks,
Filecoin DevGrants team

**Comment by pooja** [comment]
Greetings,

Thank you for your application to the Filecoin DevGrants program. We appreciate your proposal!

You have been selected for a Wave 1 DevGrant, congratulations! 🎉 

We selected grants based on:
- what makes sense to build now based on where the Filecoin node implementations are
- the likelihood of your team being able to work independently of the Filecoin engineering teams working on the core client implementations
- degree of refactoring your proposal may require as the clients evolve
- the integrity of the proposal and near-term value to the Filecoin ecosystem
- your commitment to the ecosystem and desire to maintain your project longer-term

Please reach out to the DevGrants team at devgrants@filecoin.org with a link to this PR, and we will follow up with next steps.

**Pull request closed without merging**